### ISSUE TITLE
[Accessibility]: [EAC - Migration - add user mailboxes]: Luminosity ratio is 1.425:1 for Email address in "Enter display names of mailboxes you wish to migrate" edit field which is less than 4.5:1.

### ISSUE THREAD

**Issue by GovindaSharma0310** [microsoft/fluentui#21921]
Environment Information
**Package version(s)**: "@fluentui/react": "^7.129.0",
Browser  Microsoft Edge Version 98.0.1100.3 (Official Build) dev (64-bit) 
OS versions: Chrome 98.0.4758.101, Windows 12 

Describe the issue: 
Color Contrast fails for Secondary Text under Primary when the selected people picker is clicked and the background gets blue. The Primary text color changes to white but the secondary text color remains the same due to which the Luminosity ratio is less than 4.5:1

Please provide a reproduction of the issue in a codepen:

Actual behavior: Luminosity ratio is 1.425:1 for Email address in "Enter display names of mailboxes you wish to migrate" edit field which is less than 4.5:1. 

Expected behavior:  Luminosity ratio should be greater than or equal to 4.5:1 for Email address on focus in "Enter display names of mailboxes you wish to migrate" edit field.

User Experience
If a minimum luminosity contrast ratio of 4:5:1 is not provided between text and its background, it would be difficult  people with moderately low vision to read the text. Good luminance contrast accommodates diverse color deficiencies and provides contrast that is independent of color perception.

Documentation describing expected behavior
...
